src/Helper/Product/Utility.php

```diff
@@ -94,38 +91,47 @@ class Utility
     protected $filter;
     /** @var CoreInterface */
     protected $sgCore;
+    /** @var var GetStockIdForCurrentWebsite */
+    protected $getStockIdForCurrentWebsite;
+    /** @var GetStockItemDataInterface */
+    private $getStockItemData;
```

Comment:
any specific reason this one is the only private one? There is no right or wrong here, should they all be private? Should they call be protected? CHeck other ones, I think Rainer was complaining at some point about them being all private or something.